plugins/orchestrator-leaderboard/frontend/src/globals.css

```
@@ -102,29 +102,62 @@
   .stat-card:hover {
     border-color: rgba(255, 255, 255, 0.2);
   }
+  /* Capability / model tags — matches dashboard orchestrator model pills */
+  .section-label-icon {
+    @apply p-1 rounded-md shrink-0;
+
+    background: var(--bg-tertiary);
+
+    color: #34d399;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)